RUNNING_A_PROJECT.md

```diff
@@ -7,9 +7,16 @@ As a project manager you are expected to be available in the Slack channel for y
 
 We require the following:
 
+* We adhere to an *agile methodology* that includes these components:
+  - **There is a project story/tracking board** (see note below) that is maintained by the product owner (if there is one for the project) or the project manager/lead otherwise.  This provides a focal point for sprint planning and status tracking.
+  - **Scrums (aka standups) take place on a regular schedule** - every weekday, 3 times a week, etc. - whatever meets the needs of the particular project.  These are time-boxed (15 minutes) and follow the usual routine (What have you done since last scrum?, What will you be working on?, Do you have any blockers?).  It is normal for topics to be brought up during the scrum, which are placed in the "parking lot" to be discussed after the scrum concludes.  Also, the scrums serve as a good launching point for pair programming activity.
+  - **Development takes place in sprints**, the duration of which are particular to each project, but are typically one or two weeks in length.
```

Comment:
interesting point on this - we've dropped sprints from WSO and LocalSupport, going for a more fluid ongoing work on tickets and having discussions on tickets operate asynchronously via slack.

I am concerned that we don't reflect enough - something that would often happen at the end of a sprint, but with people dropping in and out, and no one enjoying meetings (or just being unable to schedule them), I'm thinking of some kind of asynchronous ongoing reflection might be a better match for some of our projects ...